app/views/finders/_sort_options.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="govuk-form-group">
     <label for="order" class="result-region-header__sort-label govuk-label">Sort by</label>
     <select
-      class="js-order-results govuk-select"
+      class="js-order-results govuk-select result-region-header__sort-select"
```

Comment:
A lot seems to happen with this element and it gets harder to understand what and where it belongs to by looking at the markup. The newly introduced element selector (`result-region-header__sort-select`) has no associated block selector (as in a parent with a `result-redion-header` class to group it along with `result-region-header__counter`). What it actually does is modifying the font-size based on the viewport size — which could potentially be an app-level modifier for the select (e.g. `app-select--s`). Same for the `result-region-header__sort-label` above.

Reply:
added block `sort-options` and then `__label` and `__select ` elements